lib/ssl/src/ssl_crl_cache.erl

```diff
@@ -126,7 +128,7 @@ Delete CRLs from the ssl applications local cache.
 -doc(#{since => <<"OTP 18.0">>}).
```

Comment:
The `delete(Entries) -> ok | {error, Reason}` above should be deleted. Not sure why the convertion added it, but it is wrong.

Reply:
fixed